vsphere/data_source_vsphere_host.go

```diff
@@ -0,0 +1,46 @@
+package vsphere
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/vmware/govmomi"
+)
+
+func dataSourceVSphereHost() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceVSphereHostRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type: schema.TypeString,
+				Description: "The name of the host. This can be a name or path.	If not provided, the default host is used.",
+				Optional: true,
+			},
```

Comment:
_minor_ I might be wrong, but this like it it should fail the linter?

Reply:
Not too sure what linter we are talking about here but I asked in slack :)
It does pass golint (it's in my regular toolchain) and `InternalValidate`.